package.json

```diff
@@ -31,6 +34,10 @@
     "stream",
     "record"
   ],
+  "files": [
```

Comment:
I added this because I noticed that the whole source is getting published as well, not just the necessary files, thus making the npm package needlessly bigger than it needs to be. if you don't want me to do this, I can remove it, but would highly suggest it.